frontend/src/pages/organization/BillingPage/components/BillingTabGroup/BillingTabGroup.tsx

```
@@ -16,25 +16,38 @@ const tabs = [
+      window.location.origin.includes("https://gamma.infisical.com");
+
+    const tabsFiltered = isCloudInstance
+      ? tabs
+      : [{ name: "Infisical Self-Hosted", key: "tab-infisical-cloud" }];
+
     return (
       <Tabs defaultValue={tabs[0].key}>
         <TabList>
-          {tabs.map((tab) => (
+          {tabsFiltered.map((tab) => (
             <Tab value={tab.key}>{tab.name}</Tab>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)